pyaedt/emit.py

```diff
@@ -11,12 +11,12 @@
 
 
 class Emit(FieldAnalysisEmit, object):
-    """Emit class to that exposes Emit functionalities.
+    """Emit class.
 
     .. note::
-       At the moment this object has very limited functionalities.  It
-       contains only skeleton to create empty design.  No methods
-       implemented at this stage
+       At the moment, this object has very limited functionalities.  It
+       contains only a skeleton for creating an empty design.  No methods
```

Comment:
```suggestion
       contains only a skeleton for creating an empty design. No methods
```

Reply:
I'm behind the times on the transition between double and single spaces after sentences.  Thanks for catching this.